data/pigui/baseui.lua

```diff
@@ -25,7 +25,22 @@ ui.pi_4 = pi_4
 ui.pi = pi
 
 ui.anchor = { left = 1, right = 2, center = 3, top = 4, bottom = 5, baseline = 6 }
-
+ui.fullScreenWindowFlags = ui.WindowFlags {"NoTitleBar", "NoResize", "NoMove", "NoInputs", "NoSavedSettings", "NoFocusOnAppearing", "NoBringToFrontOnFocus"}
```

Comment:
There is an ImGui flag `ImGuiWindowFlags_NoBackground` that would be perfect for this purpose and avoid the need for a `ui.withStyleColors` call. It's not yet exposed, although adding it is just an extra line in the `imguiWindowFlagsTable` in `src/lua/LuaPiGui.cpp`.

Reply:
Did it for `ui.makeFullScreenHandler`, but did not do it for the "game" handler (`ui.registerHandler("game" ... )`, because almost all modules there override the window flags and the windows become visible